home.mdx

```diff
@@ -3,51 +3,35 @@ title: Meilisearch Documentation
 description: Here you'll find everything you need to learn and create with our lightning fast search engine.
 ---
 
-# Welcome to the Meilisearch documentation
+# Documentation
 
-Here you'll find everything you need to learn and create with our lightning fast search engine.
+Explore our guides and references to learn how to use Meilisearch in your projects.
 
-<Capsule className="my-[72px]">
-  <h3 className="text-3xl">Looking for SDK documentation?</h3>
-  <p>Check out this <ButtonLink href="/docs/learn/what_is_meilisearch/sdks.html" currentColor="true" >list of official Meilisearch</ButtonLink> libraries or use the Integrations dropdown menu at the top of the screen for quick access.</p>
-</Capsule>
-
-<h2 className="text-5xl">Meilisearch news</h2>
-
-Announcing Meilisearch Cloud: [register now](https://cloud.meilisearch.com/register) or [log in here](https://cloud.meilisearch.com/login) if you already have an account.
+<Capsule>
+  ## Quick start
 
-<Divider />
-
-We just raised a 15M Series A! 🥳 Check out the [TechCrunch exclusive](https://docs.meilisearch.com/#:~:text=TechCrunch%20exclusive,opens%20new%20window) or learn more [on our blog](https://blog.meilisearch.com/meilisearch-series-a/).
+  New here? Check out the [Quick start guide](/learn/getting_started/quick_start.mdx) and learn how to launch a basic Meilisearch instance.
```

Comment:
I feel pretty confident that this will fail since I don't think inserting Markdown into JSX is supported but let's try it as a test 😎

Reply:
I'm pretty sure this works. Looking at files converted by basement, many use e.g. bold inside `<Capsule>`s:  https://meilisearch-staging.vercel.app/docs/learn/advanced/asynchronous_operations#task-queue (Note right above task queue, for example).

My understanding is that markdown is fine inside components, but fails inside props.

🟢 
```jsx
<Component>
  _Oxford comma_
</Component>
```

❌ 
```jsx
<Component propertyName="_Oxford comma_"/>
```

Reply:
Makes sense, I think you're right! Time to find out for sure 😎 🚀